### PR TITLE
[risk=low][no ticket] fix broken icons by magic

### PR DESCRIPTION
**Pull request by jmthibault79**
I don't know how or why this works.  #yolo

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/components/icons.tsx [hunk 1]
```diff
@@ -14,11 +14,14 @@ import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 
 import colors, {addOpacity} from 'app/styles/colors';
 
-import {ReactComponent as registeredTierBadge} from 'assets/icons/registered-tier-badge.svg';
```
Comment (jmthibault79) [inline ui/src/app/components/icons.tsx:17]: Why are these imported differently?

File: ui/src/app/components/icons.tsx [hunk 2]
```diff
@@ -139,6 +142,4 @@ export const Times = props => <Icon shape={faTimes} {...props}/>;
 const svgIcon = src => ({size = 25, ...props}) => <img style={{height: size, width: size}} src={src} {...props}/>;
 
 export const Arrow = svgIcon(arrow);
```
Comment (jmthibault79) [inline ui/src/app/components/icons.tsx:144]: these 2 like svgIcon() and the other 2 do not.

File: ui/src/app/components/icons.tsx (hunk 1 above)
Comment (als364) [reply ui/src/app/components/icons.tsx:17]: Webpack requires images to be treated this way.
https://create-react-app.dev/docs/adding-images-fonts-and-files/

**Review by als364 — APPROVED** [review]

File: ui/src/app/components/icons.tsx (hunk 1 above)
Comment (jmthibault79) [reply ui/src/app/components/icons.tsx:17]: Thanks!  What's confusing to me is that the other 2 (arrow and googleCloudLogo) work differently.

Comment (als364) [reply ui/src/app/components/icons.tsx:17]: Oh! I'm not 100% sure why it's like that. Maybe a bad merge on my part. The way that arrow and googleCloudLogo are should work for the tier badges.

**Pull request merged**